### ISSUE TITLE
Problem to install

### ISSUE THREAD

**Issue by rickyalfaray** [danillo-alvarenga/tn3-ta_finder#1]
Dear Mr Danillo Alvarenga,

Thank you for making this software.
I would like to use this software to analyze my genome sequence, but I got a problem when I tried to install it.
I could not activate the software. I write down the command below.
Could you please help me with which step I should change? 

Aside from it, since I installed Prokka in my environment which is named myprokka, I thought blast and prodigal were already installed so I don't need to install it again. Am I correct?
Thank you so much.

Kind regards,
Ricky - Japan

```
(myprokka) [Ricky@expsrv Tn3_finder]$ cd /data/Ricky/MGE/transposon_Tn_Central/Tn3+TA_finder
(myprokka) [Ricky@expsrv Tn3+TA_finder]$ ls
tn3-ta_finder.zip
(myprokka) [Ricky@expsrv Tn3+TA_finder]$ unzip tn3-ta_finder.zip
Archive:  tn3-ta_finder.zip
3c1325044d05feec2c6ac09f037445be44ce2eb2
   creating: tn3-ta_finder-master/
  inflating: tn3-ta_finder-master/LICENSE
  inflating: tn3-ta_finder-master/README.md
  inflating: tn3-ta_finder-master/Tn3+TA_finder.py
   creating: tn3-ta_finder-master/db/
  inflating: tn3-ta_finder-master/db/LICENSE
  inflating: tn3-ta_finder-master/db/T+A.faa
  inflating: tn3-ta_finder-master/db/Tn3+R.faa
  inflating: tn3-ta_finder-master/db/database copyright.txt
(myprokka) [Ricky@expsrv Tn3+TA_finder]$ cd /data/Ricky/MGE/transposon_Tn_Central/Tn3+TA_finder/tn3-ta_finder-master
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ pwd
/data/Ricky/MGE/transposon_Tn_Central/Tn3+TA_finder/tn3-ta_finder-master
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ ls
db  LICENSE  README.md  Tn3+TA_finder.py
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ echo 'export PATH=$PATH'$/data/Ricky/MGE/transposon_Tn_Central/Tn3+TA_finder/tn3-ta_finder-master >> /data/Ricky/Software/Miniconda/envs/myprokka/etc/.bashrc
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ source /data/Ricky/Software/Miniconda/envs/myprokka/etc/.bashrc
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ ls
db  LICENSE  README.md  Tn3+TA_finder.py
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ python Tn3+TA_finder.py [-h]
(myprokka) [Ricky@expsrv tn3-ta_finder-master]$ Traceback (most recent call last):
  File "Tn3+TA_finder.py", line 33, in <module>
    from Bio.Alphabet import IUPAC
  File "/data/Ricky/Software/Miniconda/envs/myprokka/lib/python3.7/site-packages/Bio/Alphabet/__init__.py", line 21, in <module>
    "Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information."
ImportError: Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://biopython.org/wiki/Alphabet for more information.

```

**Comment by rickyalfaray** [comment]
Dear Mr Danillo Alvarenga,

Finally, I solved the problem. 
I created a new environment with Python 3.4 and an older version of the Biopython library. 
I manually installed ncbi-blast+ ; prodigal ; and the biopython.
It works now.
Thank you.

May I have other questions? Now I'm working with _H. pylori_ WGS. 
First, I got a different result with the ISFinder. I set the default setting from ISFinder with a minimum e-value of 0.000001 (from [ISFinder website.](https://isfinder.biotoul.fr/blast.php) In one of my sequences, ISFinder could detect a Transposon with an e-value of 0.0. But when I tried to run with Tn3+TA_finder with the default setting, I couldn't detect any result. My sequence is MHP04 (which I got from [Pubmed](https://www.ncbi.nlm.nih.gov/nuccore/NZ_WAEG00000000.1?report=gbwithparts&log$=seqview#sequence_NZ_WAEG00000000.1). Could you please give me some suggestions? Maybe I should change some settings in Tn3+TA_finder?
Second, If I want to run for so many WGS (more than 2000 sequences), and I want to put the results from each sequence into a different directory (for example MHP04 results will automatically be collected into a newly created directory named MHP04, then MHP29 results will automatically be collected into a newly created directory named MHP29, etc.). Do you have any bash or suggestion commands that users can use?

Thank you so much.

Kind regards,
Ricky
Japan

**Comment by danillo-alvarenga** [comment]
Dear Ricky,

Regarding the different results, remember that this script was supposed to be just an aid for manual curation. Its intent is to automatically retrieve some possible targets for the users to do a closer inspection of the annotation and see if that's really what they're looking for. So I would take the annotation suggestions with a grain of salt and do a manual validation of them. That's what we did for the paper.

My advice is that you take the sequence you're evaluating and with the possible homologues from ISfinder and Tn3+TA_finder and then do direct comparisons between them to check which one seems more likely. And keep in mind that TnComp_finder focuses exclusively on composite transposons, so you'll only get a result if you have two sequences with high similarity within a certain range. If you're fairly certain you have a composite transposon, you could try to make the default parameters less stringent.

In other words, don't consider automated predictions as final and always double check using other methods.

After you finish the analysis, you can get into the results directory and move the files into separate folders with the following bash one-liner:
`for i in *.gbk; do dir=$(echo $i | sed 's/.gbk//'); mkdir $dir; mv $i $dir; done`

If your file extension is different, just replace `.gbk`.

Best wishes,
Danillo

**Comment by rickyalfaray** [comment]
Thank you so much for your advice 😊
Now I can get all the results from my data.
I also validate the results manually to make sure.

Sincerely,
Ricky
